cloudinary/uploader.py

```diff
@@ -176,6 +176,27 @@ def create_archive(**options):
     return call_api("generate_archive", params, **options)
 
 
+def download_folder(folder_path, **options):
```

Comment:
Please move it to `utils`, close to `download_archive_url` and `download_zip_url` and please reuse `download_archive_url`.